src/core/evaluator.js

```diff
@@ -1154,6 +1156,7 @@ class PartialEvaluator {
           if (value instanceof Dict) {
             isSimpleGState = false;
 
+            // eslint-disable-next-line arrow-body-style
```

Comment:
Why not remove the return here ?

Reply:
If I change this now I'll probably need to re-run all tests again, and I figure that we can "save" the bots a little bit here.
Hence I'll defer this for now, but perhaps it can be folded into a later patch (since I've noticed some other, smaller, clean-up opportunities in this file).